src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -187,6 +190,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
 
     with self.argument_context('acr task create') as c:
         c.argument('task_name', completer=None)
+        # Overwrite default shorthand of cmd to make availability for image usage
+        c.argument('cmd', options_list=['--__cmd__'])
+        c.argument('cmd_value', help="The image to create task with.", options_list=['--cmd'])
```

Comment:
Do we need to define this twice? Can't we instead move it all the way up to `az acr` group and apply to all commands? Will that break any other command .. I guess not but please validate.

Reply:
Escalated the cmd from acr/task to directly under acr.